Cannot build AgreementMaker

I have got this error, when built with ./mvnw install -DskipTests=true

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0:compile (default-compile) on project AgreementMaker-AlignmentRepair: Compilation failure
[ERROR] /Users/ellen/eclipse-workspace/agreementmaker/AgreementMaker-OSGi/AgreementMaker-AlignmentRepair/src/main/java/am/evaluation/repair/HermitReasonerTest.java:[14,35] package jdk.internal.jline.internal does not exist

My java version is "1.8.0_171"

--- a/AgreementMaker-OSGi/AgreementMaker-AlignmentRepair/src/main/java/am/evaluation/repair/HermitReasonerTest.java
+++ b/AgreementMaker-OSGi/AgreementMaker-AlignmentRepair/src/main/java/am/evaluation/repair/HermitReasonerTest.java
@@ -1,17 +1,15 @@
 package am.evaluation.repair;
 
-import java.io.File;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.List;
-import java.util.PriorityQueue;
-import java.util.Set;
-
-import jdk.internal.jline.internal.TestAccessible;
+import am.app.mappingEngine.Mapping;
+import am.app.mappingEngine.Mapping.MappingRelation;
+import am.app.mappingEngine.similarityMatrix.SimilarityMatrix;
+import am.app.mappingEngine.utility.MatchingPair;
+import am.app.ontology.Ontology;
+import am.app.ontology.ontologyParser.OntoTreeBuilder;
+import am.output.similaritymatrix.SimilarityMatrixOutput;
+import am.utility.referenceAlignment.AlignmentUtilities;
+import am.utility.referenceAlignment.MappingsOutput;
+import com.clarkparsia.owlapi.explanation.BlackBoxExplanation;
 import org.apache.log4j.xml.DOMConfigurator;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
@@ -37,17 +35,16 @@ import org.semanticweb.owlapi.model.OWLOntologyManager;
 import org.semanticweb.owlapi.model.OWLOntologyStorageException;
 import org.semanticweb.owlapi.reasoner.Node;
 
-import am.app.mappingEngine.Mapping;
-import am.app.mappingEngine.Mapping.MappingRelation;
-import am.app.mappingEngine.similarityMatrix.SimilarityMatrix;
-import am.app.mappingEngine.utility.MatchingPair;
-import am.app.ontology.Ontology;
-import am.app.ontology.ontologyParser.OntoTreeBuilder;
-import am.output.similaritymatrix.SimilarityMatrixOutput;
-import am.utility.referenceAlignment.AlignmentUtilities;
-import am.utility.referenceAlignment.MappingsOutput;
-
-import com.clarkparsia.owlapi.explanation.BlackBoxExplanation;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.PriorityQueue;
+import java.util.Set;
 
 /**
  * @author Renu Srinivasan 

--- a/AgreementMaker-OSGi/AgreementMaker-Core/pom.xml
+++ b/AgreementMaker-OSGi/AgreementMaker-Core/pom.xml
@@ -160,6 +160,12 @@
         </dependency>
 
         <dependency>
+            <groupId>xml-apis</groupId>
+            <artifactId>xml-apis</artifactId>
+            <version>1.4.01</version>
+        </dependency>
+
+        <dependency>
             <groupId>net.sf.jwordnet</groupId>
             <artifactId>jwnl</artifactId>
             <version>1.4_rc3</version>

